src/main/java/com/thevoxelbox/voxelsniper/brush/type/stencil/StencilBrush.java

```diff
@@ -83,7 +90,7 @@ public void handleCommand(String[] parameters, Snipe snipe) {
             this.pasteOption = pasteOption;
             try {
                 this.filename = parameters[1 + pasteParam];
-                File file = new File(VoxelSniperPlugin.getPlugin().getDataFolder() + "/stencils/" + this.filename + ".vstencil");
+                File file = new File(plugin.getDataFolder() + "/stencils/" + this.filename + ".vstencil");
```

Comment:
Maybe do it WE-like and create a folder on initial setup if it doesn't exist, so the user knows where to put the files.